integreat_cms/summ_ai/utils.py

```diff
@@ -0,0 +1,106 @@
+# import os
+import logging
+import requests
+
+# from django.conf import settings
+from django.apps import apps
+from django.contrib import messages
+from django.utils.translation import ugettext as _
+
+logger = logging.getLogger(__name__)
+
+# pylint: disable=too-few-public-methods
+class SummAiAPI:
```

Comment:
```suggestion
class SummAiApiClient:
```
To be extra nit-picky, this class is the client to interact with the API, not the API itself :grin:
Also, since this file only contains the class, I would rename it from `utils.py` to `summ_ai_api_client.py`.